pkg/opensearch/client/search_request.go

```diff
@@ -411,6 +447,81 @@ func (b *aggBuilderImpl) Filters(key string, fn func(a *FiltersAggregation, b Ag
 	return b
 }
 
+func (b *aggBuilderImpl) TraceList() AggBuilder {
```

Comment:
This sets up the "aggs" object of the query to OpenSearch. See the snapshot [here](https://github.com/grafana/opensearch-datasource/pull/284/files#diff-a0ae264a941e1be77e2c5deab9b0e33792443dee84d614daa9b009fe4717e786R42).